ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent-SDO-nonprod.json

```diff
@@ -0,0 +1,23 @@
+[
```

Comment:
Can you please exclude the `*SDO-nonprod.json` in the following CCD and civil-service locations:
https://github.com/hmcts/civil-service/blob/master/Jenkinsfile_CNP#L99
https://github.com/hmcts/civil-ccd-definition/blob/master/bin/build-release-ccd-definition.sh#L10 (lines 10, 12 + 14)

This is to ensure pipelines will not break when this change is merged, as it is fully disabled

Reply:
3 small questions here Harry,

1. Within the first file you linked, does it need to be added [here](https://github.com/hmcts/civil-service/blob/master/Jenkinsfile_CNP#L130) too for aat? 
2. Do we not need to add this in for any of the Camunda files?
2. I thought the `*-nonprod.json` would cover any files ending in `-nonprod.json` which would include `blabla-SDO-nonprod.json`. Is that not the case? does the `-` before `SDO` mean that it requires a separate exclude?

Reply:
@Gulam-Hussain you are correct. The format you have used is different from other teams - so you should be fine as the standard -nonprod.json should catch these files!